src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -26,10 +29,24 @@ export const GAME_OPTION_NAMES = {
  */
 export const GAME_OPTION_VALUES = [
     {
+        name: GAME_OPTION_NAMES.THEME,
+        defaultValue: 'default',
+        description: 'Scope Theme',
+        type: 'select',
+        onChangeEvent: EVENT.SET_THEME,
```

Comment:
is this `onChangeEvent` key new?

**stopping point. will continue from here later tonight**